debs/Debian/amd64/build.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
Remove this file

Reply:
https://github.com/wazuh/wazuh-packages/pull/446/commits/f32afb3cfc9b039a7ea315e39874efab1fdcffab